imports/ui/case/case-messages.jsx

```diff
@@ -274,22 +278,31 @@ class CaseMessages extends Component {
 
   renderInputControls () {
     const { message } = this.state
+
     return (
-      <div className={[styles.inputRow, 'flex items-center overflow-visible'].join(' ')}>
-        <IconButton style={attachmentButtonStyle}>
+      <div className={[styles.inputRow, 'flex items-end overflow-visible'].join(' ')}>
+        <IconButton style={attachmentButtonStyle2}>
           <label>
             <ContentAdd color={colors.main} />
             <input type='file' className='dn' onChange={this.handleFileSelection.bind(this)} />
           </label>
         </IconButton>
         <inviteUserIcon />
         <div className='flex-grow relative'>
-          <input type='text' placeholder='Type your response' ref='messageInput'
-            onChange={this.handleMessageInput.bind(this)} value={message}
-            onKeyPress={event => { if (event.key === 'Enter' && message.replace(/\s/g, '').length > 0) { this.handleCreateMessage(event) } }}
-            className='input-reset bg-white br-pill ba b--moon-gray lh-input h2 ph3 dib outline-0 w-100' />
+          <TextField
+            id='chatbox'
+            hintText='Type your response'
+            underlineShow={false}
+            textareaStyle={whiteTextInputStyle}
+            multiLine
+            rowsMax={4}
+            fullWidth
+            value={message}
+            onChange={this.handleMessageInput.bind(this)}
+            ref='messageInput'
+          />
         </div>
-        <div className='mh2'>
+        <div className='mh2' style={{paddingBottom: '0.75rem'}}>
```

Comment:
Try to keep it consistent in the way styling is applied to elements:
* Regular DOM- use functional CSS (Tachyons) and CSS modules (where necessary)
* MUI components- functional CSS if possible, but mostly JS style objects (not defined inline)

In any case, avoid injecting inline styles altogether.
For this particular case- you could've achieved the same result if you changed 'mh2' to 'mt2 mb3 pb1', but what I would really prefer to see is that we remove vertical margins and paddings  (or set to 0 to override existing) as much as possible from the 3 elements on this row: attachment button, text field and send button, and let their container take care of most of the spacing. Only then some spacing should be applied to the elements to get them inline if necessary